styles/components/Search.scss

```diff
@@ -2,51 +2,41 @@
 // the breakpoints to be hit and the elements to be flexed at the same time
 $breakpoint-margin: 35px;
 
-.search-heading {
-  @include space-stack-m;
-}
 .search-subheading {
   display: flex;
   justify-content: space-between;
   flex-flow: column;
 
-  @media (min-width: $breakpoint-medium + $breakpoint-margin) {
+  @media (min-width: (var(--nypl-breakpoint-medium) + $breakpoint-margin)) {
```

Comment:
any reason why we're using this here but `800px` in `feedback.scss` ?

Reply:
this doesn't actually work since vars are not allowed in media queries. I updated feedback.scss to use 600px (--nypl-breakpoint-medium) instead

Reply:
Let's file a ticket to remove hard-coded breakpoints.  If Reservoir doesn't have a way to prevent us from doing this I'd contend that it should.  But whatever that fix is, we can do it in a different ticket.